bindings/python/src/lib.rs

```diff
@@ -93,6 +93,15 @@ impl Cursor {
             Ok::<(), anyhow::Error>(())
         })?;
 
+        if stmt_is_dml && self.conn.conn.get_auto_commit() {
```

Comment:
`!`missing from `self.conn.conn.get_auto_commit()`?

Reply:
@jussisaurio The Python SQLite semantics are really weird. There's indeed an implicit interactive TX, except for DML statements, which are always committed :-/